.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+## Overview
+**TL;DR**
+<One to two sentence description of the issue you are encountering or trying to solve.>
+
+### Extra Detail
+
+#### Screenshots
+<If bug fix, please upload images of the error to better showcase the problem.>
+
+#### Reasoning
+<If new idea or direction, please create a checklist of proposed ideas or next steps. [Example](https://github.com/github/training-kit/issues/415).>
+
+### How to Contribute
+- [ ] Comment on this issue if you'd like to work/collaborate on it!
+- [ ] Fork the [repository](https://github.com/github/training-kit).
+- [ ] Clone YOUR forked version of the repository to your command line. `git clone YOURURL`
+- [ ] Make changes.
```

Comment:
Are we assuming they understand how to `git add` and `git commit` in these instructions? We could change 'Make changes' to something like, 'Commit your changes. Don't know what a commit is? Use this course [Intro to GitHub OD course] to get up to speed on Git and GitHub.'